examples/run/cpu/seeding_example_greedy_ambiguity_resolution.cpp

```diff
@@ -0,0 +1,305 @@
+/** TRACCC library, part of the ACTS project (R&D line)
```

Comment:
Can you do add the ambiguity solver to `seeding_example.cpp` instead of making a new file?

Reply:
Done! I added the `perform-ambiguity-resolution` option and merged my `seeding_example_greedy_ambiguity_resolution.cpp` file into `seeding_example.cpp`